.github/workflows/integration_tests.yml

```diff
@@ -57,8 +57,10 @@ jobs:
 
       - name: Install test dependencies
         working-directory: integration_tests/mock
-        run: (test -f requirements.txt && pip install -r requirements.txt) || echo "No requirements.txt"
+        run: |
+          pip install behave
```

Comment:
should this be a dev dependency to be able to run them locally?

Reply:
I just saw it is a dev dependency. Do we need to pip install it still?

Reply:
We don't install dev dependencies in this workflow and I don't think that we have to.